### PR TITLE
Feed card for Image Tags

### PR DESCRIPTION
**Pull request by sharvaniharan**
https://phabricator.wikimedia.org/T244866

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -240,6 +240,10 @@
         <activity android:name=".onboarding.SuggestedEditsOnboardingActivity" />
 
         <activity android:name=".suggestededits.SuggestedEditsImageTagsOnboardingActivity" />
+        <activity
+            android:name=".suggestededits.SuggestedEditsFeedCardImageTagActivity"
+            android:launchMode="singleTask"
```
Comment (dbrant) [inline app/src/main/AndroidManifest.xml]: What's the reasoning for making this `singleTask`?

File: app/src/main/java/org/wikipedia/feed/suggestededits/SuggestedEditsFeedClient.kt [hunk 2]
```diff
@@ -32,6 +33,7 @@ class SuggestedEditsFeedClient(private var action: DescriptionEditActivity.Actio
     private var targetSummary: SuggestedEditsSummary? = null
     private val langFromCode: String = app.language().appLanguageCode
     private val langToCode: String = if (app.language().appLanguageCodes.size == 1) "" else app.language().appLanguageCodes[1]
+    private var page: MwQueryPage? = null
```
Comment (dbrant) [inline app/src/main/java/org/wikipedia/feed/suggestededits/SuggestedEditsFeedClient.kt]: Does `page` need to be a member variable here? Couldn't it just be a parameter of `toSuggestedEditsCard()`?

File: app/src/main/AndroidManifest.xml (hunk 1 above)
Comment (sharvaniharan) [reply app/src/main/AndroidManifest.xml]: Done

File: app/src/main/java/org/wikipedia/feed/suggestededits/SuggestedEditsFeedClient.kt (hunk 2 above)
Comment (sharvaniharan) [reply app/src/main/java/org/wikipedia/feed/suggestededits/SuggestedEditsFeedClient.kt]: Done

**Comment by sharvaniharan** [comment]
Changes done @dbrant

**Pull request merged**